### PR TITLE
enable parallel testing for appveyor

### PR DESCRIPTION
**Pull request by tphoney**

### CODE REVIEW

**Comment by tphoney** [comment]
@glennsarti @jpogran  what do you think ?

**Comment by glennsarti** [comment]
Yup that's you do it.  You'd probably want to update sync.yml otherwise modsync will overwrite it.  Example is in chocolatey (https://github.com/puppetlabs/puppetlabs-chocolatey/blob/master/.sync.yml#L45-L46)

**Comment by tphoney** [comment]
question being, is this something we want to roll out to other modules / all modules ?
or stage it

**Comment by tphoney** [comment]
@glennsarti changed to use release_check ala travis once this goes green we can look at modulesync

File: appveyor.yml [hunk 1]
```diff
@@ -42,7 +42,7 @@ build: off
 test_script:
 - bundle exec puppet -V
 - ruby -v
-- bundle exec rake spec SPEC_OPTS='--format documentation'
+- bundle exec rake release_check SPEC_OPTS='--format documentation'
```
Comment (glennsarti) [inline appveyor.yml]: typo - Should be `bundle exec rake release_checks`

Comment (tphoney) [inline appveyor.yml]: yeah :( lateness does not equal greatness...

**Comment by glennsarti** [comment]
@tphoney If you want to rerun jobs, you can just do that from the AppVeyor console

**Pull request closed without merging**